### PR TITLE
chore: remove term 'enhanced staff grader' from button

### PR DESCRIPTION
**Pull request by jansenk**
**TL;DR -** [Make the button not say "Enhanced Staff Grader" ]

JIRA: [AU-640]( https://2u-internal.atlassian.net/browse/AU-640)

**What changed?**

- "Enhanced Staff Grader" -> "Grade Available Responses"

**Developer Checklist**

- [ ] Reviewed the [release process](https://github.com/edx/edx-ora2/blob/master/.github/release_process.md)
- [ ] Translations and JS/SASS compiled
- [ ] Bumped version number in [setup.py](https://github.com/edx/edx-ora2/blob/a62e81a9b0d89223476967ec3c27f3557a850735/setup.py#L39) and [package.json](https://github.com/edx/edx-ora2/blob/a62e81a9b0d89223476967ec3c27f3557a850735/package.json#L3)

**Reviewer Checklist**

Collectively, these should be completed by reviewers of this PR:

- [ ] I've done a visual code review
- [ ] I've tested the new functionality

FYI: @openedx/content-aurora

### CODE REVIEW

**Review by nsprenkle — APPROVED** [review]
:shipit: (with the obligatory bump the versions note)
